consumer/consumer.go

```diff
@@ -385,6 +404,7 @@ func (c *Consumer) consumeState(ch AMQPChannel, queue string, connCloseCh <-chan
 
 	state := c.notifyReady(queue)
 
+
```

Comment:
But but but.... 🙈

Reply:
I thought I removed all waste. I will remove it 👌🏼